### ISSUE TITLE
No Notifications 

### ISSUE THREAD

**Issue by tompatulpan** [pixelfed/mobile-app#13]
General comment/missing feature on notifications.

No notifications within app and no notifications on iPhone for new posts or messages.

**Comment by MrAn0nym** [comment]
Rn this is intended. You could probably use this issue as a feature request though, since afaik this was only discussed on the discord server

**Comment by dansup** [comment]
Do you mean push notifications?

Navigate to Settings -> About -> Instance details -> tap and hold the domain at the bottom of the screen to copy debug data to your clipboard and paste it here please

**Comment by tompatulpan** [comment]
yeah, push notifications, but also some notifications within app to tell that there is a new message as an example (icon with a  "dot"?).
`{
    "meta": {
        "version": 1,
        "timestamp": "2022-10-09T13:50:45.738Z"
    },
    "server": {
        "instance": "pix.domain.tld",
        "version": "2.7.2 (compatible; Pixelfed 0.11.4)",
        "xid": "483557175849197575"
    },
    "client": {
        "appId": "com.pixelfed",
        "version": "1.0.0",
        "platform": {
            "device": "iPhone",
            "name": "iOS",
            "version": "15.7"
        }
    }
}`

**Comment by alexleonard** [comment]
I've been a little confused by this as well. I'm still using pixeldroid just to get notifications but prefer the design and layout of the official pixelfed app.

Are notifications planned?

**Comment by LM1LC3N7** [comment]
Same here, the official app is best in term of design but:
1. No push notifications
2. Photos are heavily cropped
